src/android/java/io/jxcore/node/BtToRequestSocket.java

```diff
@@ -40,9 +38,7 @@
 
     private int mHTTPPort = 0;
 
-    final String TAG = "--BtCon-CLIENT-Socket";
-
-    boolean mStopped = false;
+    private boolean mStopped = false;
 
     public BtToRequestSocket(BluetoothSocket socket, BtSocketDisconnectedCallBack handler,ReadyForIncoming callback) {
         print_debug("Creating BtConnectedRequestSocket");
```

Comment:
Ok, now I understood that its was not the run, but the constructor you were referring to. I moved the code to Run, as I think I did with the BtToServerSocket already earlier.